app/src/main/java/org/simple/clinic/teleconsultlog/medicinefrequency/MedicineFrequencyBottomSheet.kt

```diff
@@ -0,0 +1,131 @@
+package org.simple.clinic.teleconsultlog.medicinefrequency
+
+import android.app.Activity
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import android.util.Log
+import com.jakewharton.rxbinding3.view.clicks
+import io.reactivex.Observable
+import io.reactivex.rxkotlin.ofType
+import kotlinx.android.synthetic.main.sheet_medicine_frequency.*
+import org.simple.clinic.ClinicApp
+import org.simple.clinic.R
+import org.simple.clinic.mobius.MobiusDelegate
+import org.simple.clinic.teleconsultlog.drugduration.DrugDuration
+import org.simple.clinic.teleconsultlog.medicinefrequency.di.MedicineFrequencyComponent
+import org.simple.clinic.util.unsafeLazy
+import org.simple.clinic.widgets.BottomSheetActivity
+import javax.inject.Inject
+
+class MedicineFrequencyBottomSheet : BottomSheetActivity(), MedicineFrequencySheetUiActions {
```

Comment:
Can we rename this to `MedicineFrequencySheet` to be more consistent with other sheet naming, please?